test/algorithms/set_operations/sym_difference/Jamfile

```diff
@@ -18,5 +18,6 @@ test-suite boost-geometry-algorithms-sym_difference
     :
     [ run sym_difference_areal_areal.cpp   : : : : algorithms_sym_difference_areal_areal ]
     [ run sym_difference_linear_linear.cpp : : : : algorithms_sym_difference_linear_linear ]
+    [ run sym_difference_gc.cpp            : : : : algorithms_sym_difference_gc ]
```

Comment:
There is no unit test for `difference` yet?
That would also be useful (now, or in a new PR)

Reply:
There should be one in `set_operations/difference` directory. 

On my side the line bellow `sym_difference_tupled` looks with wrong indentation. 
Also, how do we sort the contents in those Jamfiles they seem arbitrary to me.

Reply:
Yes, `difference` tests are in different directory.